reference/managed-delivery/resource-status/index.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: single
+title:  "Resource Status"
+sidebar:
+  nav: reference
+---
+
+{% include toc %}
+
+*If you're not sure what a Managed Resource is, check out our [overview of Managed Delivery](/reference/managed-delivery) first.*
+
+## Overview
+Managed Resources will always have a **status** which describes the current state of the resource from Spinnaker's perspective. A resource's status can help you quickly answer questions like:
+  - Is Spinnaker taking automatic action on this resource right now?
+  - Does the actual configuration of this resource match the declarative configuration I gave to Spinnaker?
```

Comment:
I prefer “actual state” to “actual configuration” — I think it’s more accurate as well.

Reply:
very yes, thanks for that catch and will change. I've been trying to be disciplined about that term and as I expected writing this many words at once got me tripped up 😄

fwiw I have been making an attempt to stick to three distinct terms for the different states/configurations being talked about:
- **Declarative configuration**: the YAML or JSON you write and submit to us, might be abbreviated or delegate some decisions to defaults
- **Desired State**: the _fully realized_ settings for the thing you submitted a declarative configuration for —  all defaults filled in etc.
- **Actual state**: the settings in the real world

one of my todos that i don't have time for right now is to lay out the concept of continuous management and define those terms in a 'concepts' doc that sits parallel to these — you'll notice right now I'm making the executive decision to use 'declarative configuration' where it'd technically be more accurate to say 'desired state' because IMO we need to have a nice way to link out to a definition for desired state before we throw the term around (it feels a bit tough to parse if you're new/confused).